src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -230,6 +232,8 @@
  * @author Kohsuke Kawaguchi
  * @since 1.166
  */
+@SuppressFBWarnings(value = "UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD", 
+        justification = "This class exposes public fields as a part of its API")
```

Comment:
I would favour this on all existing public fields so that nobody can add more fields that are public. In general everywhere there is a public field as part of the API it has been a mistake